src/agni/server/sender/StatusSender.java

```diff
@@ -11,7 +11,7 @@
     	AWAY
     }
 
-    public StatusSender(I_MessageSender messageSender) {
+    public StatusSender(MessageSender messageSender) {
```

Comment:
How  come you changed this? This won't really affect any behaviour or make it any easier to code. But it does break existing tests that we can still use but not invest any time into.